tests/check_consistent.py

```diff
@@ -0,0 +1,29 @@
+import os
+import filecmp
+
+consistent_files = [
```

Comment:
Wouldn't this make more sense as a list of sets, where every member of the set should be identical?